packages/message-kit/src/lib/skills.ts

```
@@ -0,0 +1,261 @@
+          return acc;
+        }, []);
+
+        if (addressParts.length > 0) {
+          values.params[param] =
+            addressParts.length === 1 ? addressParts[0] : addressParts;
+          valueFound = true;
+        }
+      } else if (type === "number") {
+        // Handle comma-separated numbers
+        const numberParts = parts.reduce<number[]>((acc, part, idx) => {
+          if (!usedIndices.has(idx) && !isNaN(parseFloat(part))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)